frontend/src/services/publicService.js

```diff
@@ -0,0 +1,14 @@
+import ApiService from '@/common/apiService'
```

Comment:
As for backend components, let's align the services functionally.

Reply:
do you mean: instead of having publicService, we should just add getSystemMessages function to lookupService --> match with backend component?

Reply:
Yes, if you've utilized lookup.js in the backend, let's use lookupService in the frontend.

Reply:
it's updated